tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -116,16 +127,8 @@ ifneq ($(filter cmsis-nn,$(ALL_TAGS)),)
       $(CMSIS_PATH)CMSIS/DSP/Include/dsp/fast_math_functions.h \
       $(CMSIS_PATH)CMSIS/DSP/Include/dsp/filtering_functions.h \
       $(CMSIS_PATH)CMSIS/DSP/Include/dsp/interpolation_functions.h \
-      $(CMSIS_PATH)CMSIS/DSP/Include/dsp/matrix_functions.h
-
-
-    # Need to add the CMSIS Core includes path.
-    # All other CMSIS header files are included with their relative path
-    # in the CMSIS-NN micro kernel source files in
-    # tensorflow/lite/micro/kernels/cmsis-nn
-    INCLUDES += \
-      -I$(CMSIS_PATH)/CMSIS/Core/Include \
-      -I$(CMSIS_PATH)/CMSIS/DSP/Include \
-      -I$(CMSIS_PATH)/CMSIS/NN/Include
+      $(CMSIS_PATH)CMSIS/DSP/Include/dsp/matrix_functions.h \
+      $(CMSIS_PATH)CMSIS/Core/Include/cmsis_compiler.h \
+      $(CMSIS_PATH)CMSIS/DSP/Include/arm_helium_utils.h
```

Comment:
why are these additional headers needed?

Reply:
Without these the INCLUDE is needed as was there before. So those files are needed with TAGS=cmsis-nn.